src/azure-cli/azure/cli/command_modules/hdinsight/custom.py

```diff
@@ -139,6 +142,38 @@ def create_cluster(cmd, client, cluster_name, resource_group_name, cluster_type,
         raise CLIError('Either the kafka client group id and kafka client group name should be specified, '
                        'or none of them should be')
 
+    # Validate and initialize autoscale setting
+    autoscale_configuration = None
+    if autoscale_type == 'Load':
+        if not all([autoscale_min_count, autoscale_max_count]):
+            raise CLIError(
+                'When the autoscale type is Load, both of the autoscal min count and max count should be specified.')
```

Comment:
>autoscal [](start = 62, length = 8)

autoscale?

Reply:
changed it to  "both --autoscale-min-count and --autoscale-max-count"

---
In reply to: [466344116](https://github.com/Azure/azure-cli/pull/14692#discussion_r466344116) [](ancestors = 466344116)

Reply:
changed it to --autoscale-min-workernode-count and --autoscale-max-workernode-count

---
In reply to: [466795649](https://github.com/Azure/azure-cli/pull/14692#discussion_r466795649) [](ancestors = 466795649,466344116)